packages/mendel-pipeline/src/main.js

```diff
@@ -19,7 +20,17 @@ class Mendel {
 
     run(callback) {
         this.daemon.run(error => {
-            if (error) return callback(error);
+            if (error) {
+                if (error instanceof ReferenceError) {
+                    console.log(chalk.yellow([
+                        'Instance of Mendel daemon may be running.',
+                        'Attemping to recycle...',
+                    ].join('\n')));
+                } else {
+                    console.error('Unknown daemon execution error: ', error);
+                    process.exit(1);
+                }
+            }
```

Comment:
Also like that you are adding a lot more info for us on the logs. This makes easier to debug. Wondering if `debug()` would be better.